Makefile

```diff
@@ -4,13 +4,13 @@ IMAGE_NAME ?= cloud-resource-operator
 OPERATOR_IMG = $(IMAGE_REG)/$(IMAGE_ORG)/$(IMAGE_NAME):v$(VERSION)
 MANIFEST_NAME ?= cloud-resources
 NAMESPACE=cloud-resource-operator
-PREV_VERSION=0.32.0
-VERSION=0.32.1
+PREV_VERSION=0.32.1
+VERSION=0.32.2
```

Comment:
should this be a minor rather that a patch ?

Reply:
I suggested patch because it's one bug fix around the labelling usage. We don't change anything external or add any additional functionality in this release.

Reply:
Ok Patch is fine